Dockerfile.pilot-elasticsearch

```diff
@@ -1,5 +1,5 @@
-FROM alpine:3.6
+FROM docker.elastic.co/elasticsearch/elasticsearch:5.6.2
 
 ADD navigator-pilot-elasticsearch_linux_amd64 /pilot
-
+VOLUME /etc/pilot/elasticsearch/config
```

Comment:
The container here is used as a delivery mechanism for our pilot binary and should *not* contain elasticsearch itself.

Given the complex nature of navigator, it might be better to test locally against Minikube, as you'll then have constructs such as initContainers etc that will allow you to copy the pilot binary into the elasticsearch container easily (plus is a much more accurate testing environment). If that's slow to turnaround, let's work on speeding it up 😄